### ISSUE TITLE
[Bug] SystemFont on IOS

### ISSUE THREAD

**Issue by AlleSchonWeg** [baskren/Forms9Patch#81]
### Description
The System Font on iOS is not working. It falls back to default. It is easy to see with the g char.

### Steps to Reproduce

Try this simple Xaml:
```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:Forms9Patch="clr-namespace:Forms9Patch;assembly=Forms9Patch"
             x:Class="Forms9LabelFontTest.MainPage">
  <StackLayout>
    <Label Text="Normal: ggg"
           FontFamily=".SFUI"
           FontSize="40" />
    <Label Text="Semibold: ggg"
           FontFamily=".SFUI-Semibold"
           FontSize="40" />
    
    <Forms9Patch:Label FontFamily=".SFUI"
                       FontSize="40"
                       Text="Normal: ggg" />
    <Forms9Patch:Label FontFamily=".SFUI-Semibold"
                       FontSize="40"
                       Text="Semibold: ggg" />
  </StackLayout>
</ContentPage>
```
### Expected Behavior
Forms9 Label should use SFUI Font.

### Actual Behavior
Forms9 Label use default Font.

### Basic Information

- Version with issue: 2.4.7
- Last known good version: -
- IDE: VS 2019
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages: 
- Affected Devices:

### Screenshots
![Unbenannt](https://user-images.githubusercontent.com/6727366/100428639-80417a80-3094-11eb-831c-34a526258206.PNG)

**Comment by AlleSchonWeg** [comment]
Here a demo: 
[Forms9LabelFontTest.zip](https://github.com/baskren/Forms9Patch/files/5606798/Forms9LabelFontTest.zip)

**Comment by baskren** [comment]
Fixed in version 2.4.8
